c2corg_ui/static/js/feed.js

```diff
@@ -138,6 +336,60 @@ app.FeedController.prototype.getDocumentsFromFeed = function() {
   }.bind(this));
 };
 
+/**
+ * get latest topics
+ * @private
+ */
+app.FeedController.prototype.getLatestTopics_ = function() {
+  this.busyForum = true;
+  this.api.readLatestForum().then(function(response) {
+    this.handleForum(response);
+  }.bind(this), function() { // Error msg is shown in the api service
+    this.busyForum = false;
+    this.errorForum = true;
+  }.bind(this));
+};
+
+/**
+ * number cannot be < 0 and cannot be decimal
+ * @param {number} n
+ * @private
+ */
+app.FeedController.prototype.naturalNumber = function(n) {
+  if (n < 0) {
+    return 0;
+  } else {
+    if (n > 10) {
+      return 10;
+    } else {
+      return Math.round(n);
+    }
+  }
```

Comment:
When a function returns, all following blocks are not executed. As a result you could simplify the code like this:
```
if (n<0) {
  return 0;
}
if (n > 10) {
  return 10;
}
return Math.round(n);
```

Reply:
fixed